[wxwidgets] wxwidgets[core]:x64-windows -> 3.1.5#6 broken after latest changes

**Describe the bug**
Install is broken after upgrade to wxwidgets[core]:x64-windows -> 3.1.5#6

**Environment**
- OS: Win10
- Compiler: Visual Studio 2019

**To Reproduce**
Steps to reproduce the behavior:

```
C:\src\vcpkg>vcpkg install wxwidgets:x64-windows
Computing installation plan...
The following packages will be built and installed:
    wxwidgets[core]:x64-windows -> 3.1.5#6
Detecting compiler hash for triplet x64-windows...
-- Using HTTP(S)_PROXY in environment variables.
Restored 0 packages from C:\Users\jpo\AppData\Local\vcpkg\archives in 223.7 us. Use --debug to see more details.
Starting package 1/1: wxwidgets:x64-windows
Building package wxwidgets[core]:x64-windows...
-- Using cached wxWidgets-wxWidgets-9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz.
-- Extracting source C:/src/vcpkg/downloads/wxWidgets-wxWidgets-9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz
-- Applying patch disable-platform-lib-dir.patch
-- Applying patch fix-build.patch
-- Using source at C:/src/vcpkg/buildtrees/wxwidgets/src/54f4f955a1-0d7c66a59e.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: C:/src/vcpkg/packages/wxwidgets_x64-windows/share/wxwidgets/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\9a\9a8c3a297ad07d9e7d03b00cbbe2fa0541d0d9faf23340281c868c26168237a4.zip
Installing package wxwidgets[core]:x64-windows...
Elapsed time for package wxwidgets:x64-windows: 7.241 min

Total elapsed time: 7.304 min

The package wxwidgets provides CMake integration:

    find_package(wxWidgets REQUIRED)
    target_include_directories(main PRIVATE ${wxWidgets_INCLUDE_DIRS})
    target_link_libraries(main PRIVATE ${wxWidgets_LIBRARIES})
```

**Expected behavior**
wxWidgets can be used

**Failure logs** 
`2>C:\src\vcpkg\installed\x64-windows\include\wx\setup.h(2,10): fatal error C1083: Cannot open include file: '../../debug/lib//setup.h': No such file or directory
`
**Additional context**
Used to work with previous version

--- a/ports/wxwidgets/portfile.cmake
+++ b/ports/wxwidgets/portfile.cmake
@@ -102,13 +102,15 @@ if(NOT EXISTS ${CURRENT_PACKAGES_DIR}/include/wx/setup.h)
 
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
         vcpkg_replace_string("${WX_SETUP_H_FILES_REL}" "${CURRENT_PACKAGES_DIR}" "")
-        vcpkg_replace_string("${WX_SETUP_H_FILES_REL}" "${CURRENT_PACKAGES_DIR}/lib/" "")
-        vcpkg_replace_string("${WX_SETUP_H_FILES_REL}" "/setup.h" "")
+
+        string(REPLACE "${CURRENT_PACKAGES_DIR}/lib/" "" WX_SETUP_H_FILES_REL "${WX_SETUP_H_FILES_REL}")
+        string(REPLACE "/setup.h" "" WX_SETUP_H_REL_RELATIVE "${WX_SETUP_H_FILES_REL}")
     endif()
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
         vcpkg_replace_string("${WX_SETUP_H_FILES_DBG}" "${CURRENT_PACKAGES_DIR}" "")
-        vcpkg_replace_string("${WX_SETUP_H_FILES_DBG}" "${CURRENT_PACKAGES_DIR}/debug/lib/" "")
-        vcpkg_replace_string("${WX_SETUP_H_FILES_DBG}" "/setup.h" "")
+
+        string(REPLACE "${CURRENT_PACKAGES_DIR}/debug/lib/" "" WX_SETUP_H_FILES_DBG "${WX_SETUP_H_FILES_DBG}")
+        string(REPLACE "/setup.h" "" WX_SETUP_H_DBG_RELATIVE "${WX_SETUP_H_FILES_DBG}")
     endif()
 
     configure_file(${CMAKE_CURRENT_LIST_DIR}/setup.h.in ${CURRENT_PACKAGES_DIR}/include/wx/setup.h @ONLY)

--- a/ports/wxwidgets/vcpkg.json
+++ b/ports/wxwidgets/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "wxwidgets",
   "version-semver": "3.1.5",
-  "port-version": 6,
+  "port-version": 7,
   "description": [
     "Widget toolkit and tools library for creating graphical user interfaces (GUIs) for cross-platform applications. ",
     "Set WXWIDGETS_USE_STL in a custom triplet to build with the wxUSE_STL build option.",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7338,7 +7338,7 @@
     },
     "wxwidgets": {
       "baseline": "3.1.5",
-      "port-version": 6
+      "port-version": 7
     },
     "x-plane": {
       "baseline": "3.0.3",

--- a/versions/w-/wxwidgets.json
+++ b/versions/w-/wxwidgets.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9184caa631070403a5fd8c177b56907a313ad197",
+      "version-semver": "3.1.5",
+      "port-version": 7
+    },
+    {
       "git-tree": "79ef97e2255a360a5515d0fbdbb839c15bb12548",
       "version-semver": "3.1.5",
       "port-version": 6

